release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -116,6 +116,11 @@ With this release, there are several updates to the *Developer* perspective of t
 ** Cluster roles accessible to a project
 ** Actions visible on the *+Add* page
 ** Item types in the *Developer Catalog*
+* See the common updates to the *Pipeline details* and *PipelineRun details* page visualization performing the following actions:
+** Use the mouse wheel to change the zoom factor.
+** Hover over the tasks and see the task details.
```

Comment:
Hove over the tasks to see the task details.

Reply:
"... page visualization performing the following actions ..." - not clear; something seems to be missing. Depending on the context, it can be "that performs" or "by performing". I don't know. You are the best judge of the context @smunje1 .